pandas/tests/extension/base/methods.py

```diff
@@ -75,11 +77,15 @@ def test_sort_values_frame(self, data_for_sorting, ascending):
         self.assert_frame_equal(result, expected)
 
     @pytest.mark.parametrize('box', [pd.Series, lambda x: x])
-    @pytest.mark.parametrize('method', [lambda x: x.unique(), pd.unique])
+    @pytest.mark.parametrize('method', [lambda x, **kwargs: x.unique(**kwargs),
+                                        pd.unique])
     def test_unique(self, data, box, method):
         duplicated = box(data._from_sequence([data[0], data[0]]))
 
-        result = method(duplicated)
+        if isinstance(duplicated, ABCSeries) and method != pd.unique:
+            result = method(duplicated, raw=True)
+        else:
+            result = method(duplicated)
```

Comment:
This is a bit awkward, I'll admit, but the only way to avoid raising the `FutureWarning`.